src/test/groovy/com/synacy/gradprogram/spock/exercise/OrderingServiceSpec.groovy

```diff
@@ -0,0 +1,157 @@
+package com.synacy.gradprogram.spock.exercise
+
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class OrderingServiceSpec extends Specification {
+
+    def setup(){
+    }
+
+    OrderingService orderingService =  new OrderingService()
```

Comment:
This is fine but it would be much safer if we instantiate the `OrderingService` inside the `setup` method.